staging/src/k8s.io/code-generator/generate-internal-groups.sh

```diff
@@ -64,7 +64,18 @@ fi
   # To support running this script from anywhere, first cd into this directory,
   # and then install with forced module mode on and fully qualified name.
   cd "$(dirname "${0}")"
-  GO111MODULE=on go install k8s.io/code-generator/cmd/{defaulter-gen,conversion-gen,client-gen,lister-gen,informer-gen,deepcopy-gen,openapi-gen}
+  BINS=(
+      applyconfiguration-gen
+      client-gen
+      conversion-gen
+      deepcopy-gen
+      defaulter-gen
+      informer-gen
+      lister-gen
+      openapi-gen
+  )
+  # shellcheck disable=2046 # printf word-splitting is intentional
+  GO111MODULE=on go install $(printf "k8s.io/code-generator/cmd/%s " "${BINS[@]}")
```

Comment:
is a single `go install` call more efficient in a way that matters (I don't know if it shares resolution of deps, or if that is cheap enough to not matter)

Reply:
mostly just simpler